components/layout/display_list/builder.rs

```diff
@@ -236,10 +237,16 @@ impl StackingContextCollectionState {
             ),
         };
 
+        let mut stacking_context_info = FnvHashMap::default();
```

Comment:
Consider using the ```FastHashMap``` that WR uses here - I found Fnv to be significantly slower than the WR implementation. (not necessary in this PR)

Reply:
Okay. That sounds like a good idea.